client/securedrop_client/logic.py

```diff
@@ -889,8 +896,14 @@ def on_reply_download_success(self, uuid: str) -> None:
         Called when a reply has downloaded.
         """
         self.session.commit()  # Needed to flush stale data.
-        reply = storage.get_reply(self.session, uuid)
-        self.reply_ready.emit(reply.source.uuid, reply.uuid, reply.content)
+        try:
+            reply = storage.get_reply(self.session, uuid)
+            self.reply_ready.emit(reply.source.uuid, reply.uuid, reply.content)
+        except storage.SDDatabaseError as e:
+            # This shouldn't happen; if it's been downloaded successfully it should
+            # be in the database.
+            logger.error("Failed to find reply uuid")
```

Comment:
Same thing here, encode the comment in the log message, e.g. "Failed to find reply that was just downloaded"

Reply:
addressed in https://github.com/freedomofpress/securedrop-client/commit/138643be480dd82bb9ed24b787a04307a8d8c1ae